bitnami/kafka/templates/scripts-configmap.yaml

```diff
@@ -322,6 +322,18 @@ data:
     fi
     {{- end }}
 
+    {{- if .Values.sasl.zookeeper.user }}
+    export KAFKA_CONFIG_FILE=/config/kafka_jaas.conf
+    cat << EOF > /config/kafka_jaas.conf
+    Client {
+      org.apache.kafka.common.security.plain.PlainLoginModule required
+      username="{{ .Values.sasl.zookeeper.user }}"
+      password="zookeeper-password-placeholder";
+    };
+    EOF
+    replace_placeholder "zookeeper-password-placeholder" "$KAFKA_ZOOKEEPER_PASSWORD"
+    {{- end }}
+
```

Comment:
Hi,
  I think there is a mismatch of the paths here.
  This block generates the file `/config/kafka_jass.conf` but this file no mounted by the pod.
   To mimic what is being done to `/config/server.properties` you would need to copy the file.
   Also, have into account that `KAFKA_CONFIG_FILE` is being overridden in the next line.

Reply:
No there is no path-mismatch as the initcontainer is mounting kafka-config volume to /config. Later the statefulset is using volume kafka-config to mount server.properties and kafka_jaas.conf files to folder /opt/bitnami/kafka/config/

See https://github.com/bitnami/charts/blob/main/bitnami/kafka/templates/_helpers.tpl#L920
and for example for server.properties mounting:
https://github.com/bitnami/charts/blob/main/bitnami/kafka/templates/broker/statefulset.yaml#L302

The KAFKA_CONFIG_FILE Variable is used in function "replace_placeholder" to find the right file where we have to replace placeholder.

See https://github.com/bitnami/charts/blob/main/bitnami/kafka/templates/scripts-configmap.yaml#L140

Reply:
Hi,
could you please re-review my PR?
Thank you

Reply:
Sorry for the delay.
Let me take a look